src/store/channels-list/utils.ts

```diff
@@ -28,8 +28,8 @@ export const toLocalChannel = (input): Partial<Channel> => {
     otherMembers: input.otherMembers || [],
     lastMessage: input.lastMessage || null,
     lastMessageCreatedAt: input.lastMessage?.createdAt || null,
-    isChannel: input.isChannel === undefined ? true : input.isChannel,
-    groupChannelType: input.groupChannelType || '',
+    isChannel: (input.isChannel ?? true) || !!input.isChannel,
+    groupChannelType: input.groupChannelType || '', // Is this the right default to use?
```

Comment:
Hm shouldn't it be 'private' or 'public' as default?

Reply:
Yep, it should but it's been like this since someone wrote it so I didn't want to mix functionality changes with refactoring.